aws/resource_aws_ssm_association_test.go

```diff
@@ -532,7 +516,7 @@ func testAccCheckAWSSSMAssociationDestroy(s *terraform.State) error {
 		}
 	}
 
-	return fmt.Errorf("Default error in SSM Association Test")
+	return nil
```

Comment:
Standardizing with all other resource `CheckDestroy` functions, otherwise returns error with `_disappears` testing since there are no resource states post-apply in that scenario. See also https://github.com/hashicorp/terraform-provider-aws/issues/15847